lib/rouge/lexers/rescript.rb

```diff
@@ -0,0 +1,121 @@
+# -*- coding: utf-8 -*- #
+# frozen_string_literal: true
+
+module Rouge
+  module Lexers
+    load_lexer 'ocaml/common.rb'
+
+    class ReScript < OCamlCommon
+      title "ReScript"
+      desc "The ReScript programming language (rescript-lang.org)"
+      tag 'rescript'
+      filenames '*.res', '*.resi'
+      mimetypes 'text/x-rescript'
+
+      operators = %r([;_!$%&*+/<=>@^|-]+)
```

Comment:
Is there any reason to make this a class-level instance variable? It's only used in one place, right?

Reply:
You're right